### DISCUSSION TITLE
Why does the filter in issues in GitHub not works with the author?

### DISCUSSION THREAD

**Discussion by lalo-mx** [community/community#22312] [New to GitHub]
<p>I was trying to find an issue that i created so i went to the project issues and in Filters typed my username to find an <a href="https://github.com/IgniteUI/igniteui-angular/issues/3880" rel="nofollow noopener">issue</a>.</p>
<p>In filters i typed my username: lalo-mx. The search generated was: <a href="https://github.com/IgniteUI/igniteui-angular/issues?utf8=%E2%9C%93&amp;q=lalo-mx" rel="nofollow noopener">https://github.com/IgniteUI/igniteui-angular/issues?utf8=%E2%9C%93&amp;q=lalo-mx</a></p>
<p>The issue that is was looking for is not there.</p>
<p>Then i set the filter to author:lalo-mx and was able to locate the issue. The search generated was: <a href="https://github.com/IgniteUI/igniteui-angular/issues?utf8=%E2%9C%93&amp;q=author%3Alalo-mx+" rel="nofollow noopener">https://github.com/IgniteUI/igniteui-angular/issues?utf8=%E2%9C%93&amp;q=author%3Alalo-mx+</a></p>
<p>Why does the filter in issues in GitHub not works with the author?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/lalo-mx">@lalo-mx</a> ,</p>
<p>You can just type  <strong>author:</strong> followed by the username in the search field. So if you’d like to find issues opened by you, just type  <strong>author:lalo-mx</strong>. If you just type  <strong>lalo-mx</strong> , GitHub will search for issues that contain your username in it’s conversation - which won’t be the case for most of your issues I suppose.</p>
<p>If you’d like more information, just read the documentation over here: <a href="https://help.github.com/articles/searching-issues-and-pull-requests/" rel="nofollow noopener">https://help.github.com/articles/searching-issues-and-pull-requests/</a></p>
